source/Ship.cpp

```diff
@@ -2177,15 +2177,14 @@ shared_ptr<Ship> Ship::Board(bool autoPlunder)
 	{
 		// Take any commodities that fit.
 		victim->cargo.TransferAll(cargo, false);
-		// Stop targeting this ship.
-		SetTargetShip(shared_ptr<Ship>());
 		
 		// Pause for two seconds before moving on.
 		pilotError = 120;
 	}
 	
 	// Stop targeting this ship (so you will not board it again right away).
-	SetTargetShip(shared_ptr<Ship>());
+	if(personality.Disables() || !personality.Plunders())
+		SetTargetShip(shared_ptr<Ship>());
```

Comment:
Which AI ships will even reach this point in the code that don't have the plunders personality? The only time ships board each other are if a ship is assisting a friendly, or if a ship has the plunders personality and is boarding a hostile. The friendly assisting is handled above this and exits the function. So wouldn't it be the case that the only ships to reach this point are ships with plunders, meaning this if statement is always true for non-player ships? 
```suggestion
	if(isYours)
		SetTargetShip(shared_ptr<Ship>());
```
The comment even says so *you* will not board it again right away, as the target wasn't reset by this point if you're the one doing the boarding since autoPlunder is false for the player.

Reply:
The target is only reset here now, so it has to be reset in all cases except when the ship is `plundering` and not `disables`.